.env.example

```diff
@@ -24,6 +24,9 @@ SQL_EXPORTER_IP=127.0.0.1
 # 9399 is default from upstream project
 SQL_EXPORTER_PORT=9399
 
+# Use to customize log retention for docker containers
+#LOG_MAX_SIZE=50m
+#LOG_MAX_FILES=20
```

Comment:
FYI to potential future reviewers:
> By default, the local driver preserves 100MB of log messages per container and uses automatic compression to reduce the size on disk. The 100MB default value is based on a 20M default size for each file and a default count of 5 for the number of such files (to account for log rotation).

from https://docs.docker.com/engine/logging/drivers/local/